### PR TITLE
Switch all sgpp src includes to angle brackets

### PR DESCRIPTION
**Pull request by spc90**
I've replaced all includes of the form "sgpp/moduleName/..." to <sgpp/moduleName/...>, as should be. Only the local .hpp files in tests and/or examples folders are referenced with quotes.

### CODE REVIEW

File: optimization/tests/GridCreator.cpp [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 #include <vector>
 
+
```
Comment (leiterrl) [inline optimization/tests/GridCreator.cpp]: I suppose this was unintended?

Comment (spc90) [reply optimization/tests/GridCreator.cpp]: Sort of. I like to have an extra space between headers and code, but sure, let's mark it as unintentional :D It shouldn't matter with the formatter, but it does indeed add one extra line the reviewer has to look at... :)

Comment (leiterrl) [reply optimization/tests/GridCreator.cpp]: Alright nvm then. As long as the stylechecker doesn't complain I'm fine with it :)

**Comment by leiterrl** [comment]
Well that didn't work out. Seems like your auto-format settings are still not aligned with jenkins.
https://simsgs.informatik.uni-stuttgart.de/jenkins/job/SGpp-style-testing/246/

**Comment by spc90** [comment]
@leiterrl Nah, it's not the formatter, it's me :D I messed up with the order of includes, by "fixing" something that didn't need fixing. I'll solve this immediately.

**Pull request merged**